fastods/src/main/java/com/github/jferard/fastods/style/TableCellStyle.java

```diff
@@ -123,9 +123,11 @@ public void addToElements(final OdsElements odsElements) {
     private void appendCellProperties(final XMLUtil util, final Appendable appendable)
             throws IOException {
         appendable.append("<style:table-cell-properties");
+        
+        /*
```

Comment:
1. We should keep a way to set a background color;
2. I don't see a background color in the default cell style:

```
    /**
     * This is the default cell style: left and top align, no wrap.
     */
    public static final TableCellStyle DEFAULT_CELL_STYLE =
            TableCellStyle.builder("Default").verticalAlign(VerticalAlign.TOP).fontWrap(false)
                    .backgroundColor(SimpleColor.NONE).allMargins(Length.NULL_LENGTH)
                    .fontName(LOFonts.LIBERATION_SANS).parentCellStyle(null).build();
```
Can you check if you have a background color with the last version?

Reply:
I checked, the latest version relsoved the issue, before i was running 0.32 through jitpack on Android where the issue appeard